docs/features.md

```diff
@@ -19,7 +19,11 @@ The [Cake.Issues.DupFinder addin] provides the following features:
 | <span class="glyphicon glyphicon-remove" style="color:red"></span> | `IIssue.ProjectName`              |                                  |
 | <span class="glyphicon glyphicon-remove" style="color:red"></span> | `IIssue.ProjectFileRelativePath`  |                                  |
 | <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.AffectedFileRelativePath` |                                  |
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IIssue.Identifier`               |                                  |
```

Comment:
In the documentation of the other addins I've added `Identifier` after `ProviderName`. For sake of consistency you might order it the same.